dataverse-webapp/src/main/webapp/resources/vecler/theme-base.scss

```diff
@@ -2762,6 +2761,10 @@ table[id$="resultsTable"] {
 		> a, > span.text-muted, .card-title-icon-block + div {
 			font-size: 0.867em;
 		}
+
+		.label-container {
+			word-break: break-all;
```

Comment:
maybe instead `word-break: break-all` we could do `.label-container > .label { display: inline-block; }` it seems quite ok when rendering when I tried it. (word-break has some disadvantage because it can break to new line when some label have two words - for example "Pobieracz Plików")

Reply:
👍 You're right, that looks like a better approach. Thanks!